src/Collections/Sets/Sets.dfy

```diff
@@ -193,4 +195,36 @@ module {:options "-functionSyntax:4"} Sets {
     LemmaSubsetSize(x, range);
   }
 
+  /** Any totally ordered set contains a unique minimal element. */
+  lemma LemmaUniqueMinimum<T(!new)>(R: (T, T) -> bool, s: set<T>) returns (m: T)
```

Comment:
Where does the duplication between `src/Collections/Sets/Sets.dfy` and `src/dafny/Collections/Sets.dfy` come from?

Reply:
https://github.com/dafny-lang/libraries/pull/92